config.default.js

```diff
@@ -3,12 +3,10 @@ import fs from 'node:fs';
 let mongo = {
   // Setting the connection string will only give access to that database
   // to see more databases you need to set mongodb.admin to true or add databases to the mongodb.auth list
-  // It is RECOMMENDED to use connectionString instead of individual params, other options will be removed later.
+  // As recommended, a connection String is used instead of the individual params.
   // More info here: https://docs.mongodb.com/manual/reference/connection-string/
-  connectionString: process.env.ME_CONFIG_MONGODB_SERVER ? '' : process.env.ME_CONFIG_MONGODB_URL,
-  host: '127.0.0.1',
-  port: '27017',
-  dbName: '',
+  connectionString: process.env.ME_CONFIG_MONGODB_URL,
+  ssl: false,
```

Comment:
What  happens if you set in connection string ?ssl=true, it would be override by this setting?